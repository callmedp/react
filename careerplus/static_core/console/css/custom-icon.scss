@import "../../shinelearn/css/path";

$sprite-exe: url($path + 'images/executive/sprite-executive.png') no-repeat;

.exe-icon-apply {
	width: 27px;
	height: 30px;
	background: $sprite-exe 0 0;
	display: inline-block;
}

.exe-icon-batch {
	width: 26px;
	height: 26px;
	background: $sprite-exe -37px 0;
	display: inline-block;
}

.exe-icon-duration {
	width: 30px;
	height: 35px;
	background: $sprite-exe -75px 0;
	display: inline-block;
}

.exe-icon-batch-sm {
	width: 15px;
	height: 15px;
	background: $sprite-exe -96px -91px;
	display: inline-block;
}

.exe-icon-duration-sm {
	width: 13px;
	height: 15px;
	background: $sprite-exe -97px -112px;
	display: inline-block;
}

.exe-icon-about-program {
	width: 24px;
	height: 20px;
	background: $sprite-exe -114px 0;
	display: inline-block;
}

.exe-icon-who-attend {
	width: 29px;
	height: 20px;
	background: $sprite-exe -147px 0;
	display: inline-block;
}

.exe-icon-manager {
	width: 37px;
	height: 40px;
	background: $sprite-exe 0 -40px;
	display: inline-block;
}

.exe-icon-criteria {
	width: 20px;
	height: 20px;
	background: $sprite-exe -181px 0;
	display: inline-block;
}

.exe-icon-faculty {
	width: 37px;
	height: 22px;
	background: $sprite-exe -207px 0;
	display: inline-block;
}

.exe-icon-chevron-left {
	width: 11px;
	height: 19px;
	background: $sprite-exe -485px -40px;
	display: inline-block;
}

.exe-icon-chevron-right {
	width: 11px;
	height: 19px;
	background: $sprite-exe -471px -40px;
	display: inline-block;
}

.exe-icon-benefit {
	width: 29px;
	height: 20px;
	background: $sprite-exe -248px 0;
	display: inline-block;
}

.exe-icon-feat-profile {
	width: 44px;
	height: 40px;
	background: $sprite-exe -158px -40px;
	display: inline-block;
}

.exe-icon-feat-profile {
	width: 44px;
	height: 40px;
	background: $sprite-exe -158px -40px;
	display: inline-block;
}

.exe-icon-shine-credit {
	width: 42px;
	height: 36px;
	background: $sprite-exe -211px -40px;
	display: inline-block;
}

.exe-icon-education-provider {
	width: 40px;
	height: 40px;
	background: $sprite-exe -259px -40px;
	display: inline-block;
}

.exe-icon-0-emi {
	width: 39px;
	height: 40px;
	background: $sprite-exe -305px -40px;
	display: inline-block;
}

.exe-icon-free-service {
	width: 43px;
	height: 41px;
	background: $sprite-exe -350px -40px;
	display: inline-block;
}

.exe-icon-refer-earn {
	width: 41px;
	height: 45px;
	background: $sprite-exe -400px -40px;
	display: inline-block;
}

.exe-icon-program-structure {
	width: 28px;
	height: 22px;
	background: $sprite-exe -284px 0;
	display: inline-block;
}

.exe-icon-assesment {
	width: 20px;
	height: 25px;
	background: $sprite-exe -319px 0;
	display: inline-block;
}

.exe-icon-institute {
	width: 22px;
	height: 22px;
	background: $sprite-exe -344px 0;
	display: inline-block;
}

.exe-icon-testimonials {
	width: 25px;
	height: 25px;
	background: $sprite-exe -373px 0;
	display: inline-block;
}

.exe-icon-fee-payments {
	width: 21px;
	height: 26px;
	background: $sprite-exe -406px 0;
	display: inline-block;
}

.exe-icon-faq {
	width: 25px;
	height: 25px;
	background: $sprite-exe -433px 0;
	display: inline-block;
}

.exe-icon-brochure {
	width: 16px;
	height: 18px;
	background: $sprite-exe -484px 0;
	display: inline-block;
}

.exe-icon-request-call {
	width: 18px;
	height: 20px;
	background: $sprite-exe -462px 0;
	display: inline-block;
}

.exe-icon-place-order {
	width: 37px;
	height: 40px;
	background: $sprite-exe -119px -91px;
	display: inline-block;
}

.exe-icon-study-material {
	width: 39px;
	height: 39px;
	background: $sprite-exe -167px -91px;
	display: inline-block;
}

.exe-icon-start-learning {
	width: 37px;
	height: 42px;
	background: $sprite-exe -216px -91px;
	display: inline-block;
}

.exe-icon-get-certified {
	width: 39px;
	height: 30px;
	background: $sprite-exe -266px -91px;
	display: inline-block;
}