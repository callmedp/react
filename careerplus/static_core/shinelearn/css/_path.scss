// For local use only for desktop
//   $path : '/media/static/shinelearn/images/hr-insider/';

// For local use only for Mobile
//   $path : '/media/static/shinelearn/images/mobile/hr-insider/';

// For Production 
  // $path : '/l/s/shinelearn/';

// For local use only
$path : '/media/static/shinelearn/';
