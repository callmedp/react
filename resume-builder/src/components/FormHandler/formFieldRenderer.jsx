import React from 'react'


export const renderField = ({
                                input,
                                label,
                                type,
                                className,
                                meta: {touched, error, warning}
                            }) => (

        <input {...input} className={className}  placeholder={label} type={type}/>
        // {touched &&
        // ((error && <span className={'Error-message'}>{error}</span>) ||
        //     (warning && <span className={'Warn-Message'}>{warning}</span>))}
)

/*

export const datePicker = ({
                               input,
                               label,
                               type,
                               onDateChange,
                               meta: {touched, error, warning}
                           }) => (
    <div>
        <div className={'Top-space'}>
            <DatePicker {...input} dateFormat="yyyy-MM-dd" placeholderText={label} className="Field-spacing"
                        selected={input.value ? moment(input.value).format("YYYY-MM-DD") : null}
                        onChange={date => input.onChange(moment(date).format("YYYY-MM-DD"))}
            />
            {touched &&
            ((error && <span className={'Error-message'}>{error}</span>) ||
                (warning && <span className={'Warn-Message'}>{warning}</span>))}
        </div>
    </div>
)

export const renderSelect = ({
                                 input,
                                 label,
                                 meta: {touched, error, warning},
                                 options
                             }) => (
    <div>
        <div className={'Top-space'}>
            <Select {...input}
                    placeholder={label}
                    options={options}
                    onBlur={() => {
                        input.onBlur(input.value)
                    }}
            />
            {touched &&
            ((error && <span className={'Error-message'}>{error}</span>) ||
                (warning && <span className={'Warn-Message'}>{warning}</span>))}
        </div>
    </div>
);

const adaptFileEventToValue = delegate => e => delegate(e.target.files[0]);

export const renderFileInput = ({
                                    input: {value: omitValue, onChange, onBlur, ...inputProps},
                                    meta: omitMeta,
                                    ...props
                                }) => {
    return (
        <input
            onChange={adaptFileEventToValue(onChange)}
            onBlur={adaptFileEventToValue(onBlur)}
            type="file"
            {...props.input}
            {...props}
        />
    );
};

export const renderDynamicSelect = ({
                                        input,
                                        loadOptions,
                                        label,
                                        defaultOptions
                                    }) => (
    <AsyncSelect {...input}
                 loadOptions={loadOptions}
                 defaultOptions={defaultOptions}
                 placeholder={label}
                 isMulti={true}
                 onBlur={() => {
                     input.onBlur(input.value)
                 }}
    />
)
*/

export const renderTextArea = ({
                                   input,
                                   label,
                                   type,
                                   meta: {touched, error, warning}

                               }) => (

            <textarea {...input} placeholder={label} type={type}/>
            // {touched &&
            // ((error && <span className={'Error-message'}>{error}</span>) ||
            //     (warning && <span className={'Warn-Message'}>{warning}</span>))}


)

