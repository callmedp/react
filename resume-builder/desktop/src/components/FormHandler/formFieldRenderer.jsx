import React from 'react';
import DatePicker from "react-datepicker";
import "react-datepicker/dist/react-datepicker.css";
import AsyncSelect from 'react-select/lib/Async';
import AsyncCreatableSelect from 'react-select/lib/AsyncCreatable';
import Select from 'react-select';
import makeAnimated from 'react-select/lib/animated';
import moment from 'moment';
import {Field} from "redux-form";


export const renderField = ({
                                input,
                                label,
                                type,
                                className,
                                tillTodayDisable,
                                index,
                                text,
                                iconClass,
                                autoFocus,
                                meta: {touched, error, warning}
                            }) => {
    return (
        <React.Fragment>
            {index ?


                <div className={"Error " + (touched && error ? 'errormsg' : '')}>
                    <input {...input} className={className}
                           onClick={(e) => tillTodayDisable(index, !input.checked, e)}
                           autoComplete="off" placeholder={label} type={type}/>
                    {
                        !!(text) && <span>{text}</span>
                    }
                    {touched &&
                    ((error && <span className={'errormsg-txt'}>{error}</span>) ||
                        (warning && <span className={'Warn-Message'}>{warning}</span>))}
                </div>
                :
                <div className={"input-group " + (touched && error ? 'errormsg' : '')}>
                    <div className="input-group--input-group-icon">
                        <span className={iconClass}></span>
                    </div>
                    <div className={"Error " + (touched && error ? 'errormsg' : '')}>
                        <input {...input}
                               autoFocus={autoFocus}
                               className={className} autoComplete="off" placeholder={label} type={type}/>
                        {touched &&
                        ((error && <span className={'errormsg-txt'}>{error}</span>) ||
                            (warning && <span className={'Warn-Message'}>{warning}</span>))}
                    </div>
                </div>

            }
        </React.Fragment>
    )
};


export const datepicker =
    ({
         input,
         label,
         type,
         onDateChange,
         disabled,
         iconClass,
         maxDateAllowed,
         yearDropDownItemNumber,
         meta: {touched, error, warning}
     }) => (
        <div className={"input-group " + (touched && error ? 'errormsg' : '')}>
            <div className="input-group--input-group-icon">
                <span className={iconClass}></span>
            </div>
            <div className={"Error " + (touched && error ? 'errormsg' : '')}>
                <DatePicker {...input}
                            value={disabled ? moment().format('YYYY-MM-DD').toString() : input.value}
                            dateFormat="yyyy-MM-dd"
                            autoComplete="off"
                            selected={input.value ? new Date(input.value) : null}
                            maxDate={maxDateAllowed ? new Date() : null}
                    // minDate={dateValue !== ''  ? moment(dateValue).add(1, 'days') : null}
                            onChange={date => input.onChange(date)}
                            showYearDropdown
                            yearDropdownItemNumber={yearDropDownItemNumber}
                            scrollableYearDropdown
                            showMonthDropdown
                            disabled={disabled}
                            popperModifiers={{
                                offset: {
                                    enabled: true,
                                    offset: '5px, 10px'
                                },
                                preventOverflow: {
                                    enabled: true,
                                    escapeWithReference: false, // force popper to stay in viewport (even when input is scrolled out of view)
                                    boundariesElement: 'viewport'
                                }
                            }}
                />
                {touched &&
                ((error && <span className={'errormsg-txt'}>{error}</span>) ||
                    (warning && <span className={'Warn-Message'}>{warning}</span>))}
            </div>
        </div>
    )

const selectStyles = {menu: styles => ({...styles, zIndex: 999})};


export const renderSelect = ({
                                 input,
                                 label,
                                 meta: {touched, error, warning},
                                 options,
                                 isMulti,
                                 iconClass,
                                 closeMenuOnSelect
                             }) => (
    <div className={"input-group " + (touched && error ? 'errormsg' : '')}>
        <div className="input-group--input-group-icon">
            <span className={iconClass}></span>
        </div>
        <div className={"Error "+ (touched && error ? 'errormsg' : '')}>
            <Select {...input}
                    placeholder={label}
                    styles={{menuPortal: base => ({...base, zIndex: 9999})}}
                    menuPortalTarget={document.body}
                    options={options}
                    isMulti={isMulti}
                    closeMenuOnSelect={closeMenuOnSelect}
                    autoComplete="off"
                    menuPosition={'absolute'}
                    menuPlacement={'auto'}
                    components={makeAnimated()}
                    onBlur={() => {
                        input.onBlur(input.value)
                    }}
            />
            {touched &&
            ((error && <span className={'errormsg-txt'}>{error}</span>) ||
                (warning && <span className={'Warn-Message'}>{warning}</span>))}
        </div>
    </div>
);


export const renderDynamicSelect = ({
                                        input,
                                        loadOptions,
                                        label,
                                        defaultOptions,
                                        iconClass,
                                        isMulti,
                                        closeMenuOnSelect,
                                        meta: {touched, error, warning}
                                    }) => {
    console.log('0---', input);
    return (
        <div className={"input-group " + (touched && error ? 'errormsg' : '')}>
            <div className="input-group--input-group-icon">
                <span className={iconClass}></span>
            </div>
            <div className={"Error "+ (touched && error ? 'errormsg' : '')}>
                <AsyncSelect {...input}
                             loadOptions={loadOptions}
                             styles={{menuPortal: base => ({...base, zIndex: 9999})}}
                             menuPortalTarget={document.getElementById('right-panel-section')}
                             menuPosition={'absolute'}
                             menuPlacement={'auto'}
                             defaultOptions={defaultOptions}
                             placeholder={label}
                             isMulti={isMulti}
                             autoComplete="off"
                             closeMenuOnSelect={closeMenuOnSelect}
                             onBlur={() => {
                                 input.onBlur(input.value)
                             }}
                />
                {touched &&
                ((error && <span className={'errormsg-txt'}>{error}</span>) ||
                    (warning && <span className={'Warn-Message'}>{warning}</span>))}
            </div>
        </div>
    )
}


export const renderAsyncCreatableSelect = ({
                                               input,
                                               loadOptions,
                                               label,
                                               defaultOptions,
                                               iconClass,
                                               isMulti,
                                               closeMenuOnSelect,
                                               meta: {touched, error, warning}
                                           }) => {
    console.log('0---', input);
    return (
        <div className={"input-group " + (touched && error ? 'errormsg' : '')}>
            <div className="input-group--input-group-icon">
                <span className={iconClass}></span>
            </div>
            <div className={"Error "+ (touched && error ? 'errormsg' : '')}>
                <AsyncCreatableSelect {...input}
                                      cacheOptions
                                      loadOptions={loadOptions}
                                      styles={{menuPortal: base => ({...base, zIndex: 9999})}}
                                      menuPortalTarget={document.getElementById('right-panel-section')}
                                      menuPosition={'absolute'}
                                      menuPlacement={'auto'}
                                      defaultOptions={defaultOptions}
                                      placeholder={label}
                                      isMulti={isMulti}
                                      autoComplete="off"
                                      closeMenuOnSelect={closeMenuOnSelect}
                                      onBlur={() => {
                                          input.onBlur(input.value)
                                      }}
                />
                {touched &&
                ((error && <span className={'errormsg-txt'}>{error}</span>) ||
                    (warning && <span className={'Warn-Message'}>{warning}</span>))}
            </div>
        </div>
    )
}

export const renderTextArea = ({
                                   input,
                                   label,
                                   type,
                                   rows,
                                   iconClass,
                                   noIcon,
                                   meta: {touched, error, warning}

                               }) => (
    <React.Fragment>
        {noIcon ?
            <div className={"Error "+ (touched && error ? 'errormsg' : '')}>
        <textarea {...input}
                  autoComplete="off"
                  placeholder={label}
                  rows={rows} type={type}/>
                {touched &&
                ((error && <span className={'errormsg-txt'}>{error}</span>) ||
                    (warning && <span className={'Warn-Message'}>{warning}</span>))}
            </div>
            :
            <div className={"input-group " + (touched && error ? 'errormsg' : '')}>
                <div className="input-group--input-group-icon">
                    <span className={iconClass}></span>
                </div>
                <div className={"Error "+ (touched && error ? 'errormsg' : '')}>
        <textarea {...input}
                  autoComplete="off"
                  placeholder={label}
                  rows={rows} type={type}/>
                    {touched &&
                    ((error && <span className={'errormsg-txt'}>{error}</span>) ||
                        (warning && <span className={'Warn-Message'}>{warning}</span>))}
                </div>
            </div>
        }

    </React.Fragment>


)

