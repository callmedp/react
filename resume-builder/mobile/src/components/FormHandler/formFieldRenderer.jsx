import React from 'react'
import DatePicker from "react-datepicker";
import "react-datepicker/dist/react-datepicker.css";
import AsyncSelect from 'react-select/lib/Async';


export const renderField = ({
                                input,
                                label,
                                type,
                                className,
                                iconClass,
                                id,
                                prepend,
                                meta: {touched, error, warning}
                            }) => (

        
        <React.Fragment>
            <label className="form__label" htmlFor={input.name}>{label}</label>
            {!prepend ?
                <React.Fragment>
                    <input {...input} className={className +(error ? " error" : "")} id={id} type={type}/>
                    {touched &&
                        ((<span className={'error-message'}>{error}</span>) ||
                            (warning && <span className={'warn-Message'}>{warning}</span>))
                    }
                </React.Fragment> :
                
            <div className={"input-group " + (error ? "error" : "")} >
                <div className="input-group__prepend">
                <span className="input-group__text">
                    <i className={iconClass}></i>
                </span>
                </div>
                <input {...input} className={className} id={id} type={type}/>
                {touched &&
                    ((<span className={'error-message'}>{error}</span>) ||
                        (warning && <span className={'warn-Message'}>{warning}</span>))
                }
            
            </div>
            }
        </React.Fragment>
);


export const datepicker =
    ({
         input,
         label,
         className,
         id,
         meta: {touched, error, warning}
     }) => (
        <React.Fragment>
            <label className="form__label" htmlFor={input.name}>{label}</label>
            <div className={"input-group " + (error ? "error" : "")}>
                <div className="input-group__prepend">
                    <span className="input-group__text">
                        <i className="sprite icon--date"></i>
                    </span>
                </div>
                <DatePicker {...input} dateFormat="yyyy-MM-dd" className={className}
                        selected={input.value ? new Date(input.value) : null}
                        onChange={date => input.onChange(date)} id={id}
                />
                {touched &&  ((error && <span className={'error-message'}>{error}</span>) ||
                (warning && <span className={'warn-Message'}>{warning}</span>))}
            </div>
            
        </React.Fragment>
    )


export const renderSelect = ({
                                 input,
                                 label,
                                 meta: {touched, error, warning},
                                 children,
                                 className,
                                 iconClass,
                                 prepend
                             }) => (
    <React.Fragment>
        <label className="form__label" htmlFor={input.name}>{label}</label>
        {prepend ?
        <div className={"input-group " +(error ? "error" : "")}>
            <div className="input-group__prepend">
                <span className="input-group__text">
                    <i className={iconClass}></i>
                </span>
            </div>
            <select {...input} className={className}
                onBlur={() => {
                    input.onBlur(input.value)
                    console.log(input.label)
                }}
                >
                {children}
            </select>
            <div className="select-error">
                {touched &&
                ((error && <span className={'error-message'}>{error}</span>) ||
                    (warning && <span className={'warn-Message'}>{warning}</span>))}
            </div>
        </div> :
        <React.Fragment>



            <select {...input} className={className +(error ? " error" : "")}
                onBlur={() => {
                    input.onBlur(input.value)
                    console.log(input.label)
                }}
                >
                {children}
            </select>
                {touched &&
                ((error && <span className={'error-message'}>{error}</span>) ||
                    (warning && <span className={'warn-Message'}>{warning}</span>))}
        </React.Fragment>
        }
        
    </React.Fragment>
);


export const renderDynamicSelect = ({
                                        input,
                                        loadOptions,
                                        label,
                                        defaultOptions,
                                        meta: {touched, error, warning}
                                    }) => (
    <div className="flex-auto">
        <AsyncSelect {...input}
                     loadOptions={loadOptions}
                     defaultOptions={defaultOptions}
                     placeholder={label}
                     isMulti={true}
                     onBlur={() => {
                         input.onBlur(input.value)
                     }}
        />
        {touched &&
        ((error && <span className={'error-message'}>{error}</span>) ||
            (warning && <span className={'warn-Message'}>{warning}</span>))}
    </div>
)

export const renderTextArea = ({
                                   input,
                                   label,
                                   type,
                                   rows,
                                   className,
                                   iconClass,
                                   prepend,
                                   meta: {touched, error, warning}

                               }) => (
    <React.Fragment>
        <label className="form__label" htmlFor={input.name}>{label}</label>
        {prepend ?
        <div className="input-group">
            <div className="input-group__prepend">
                <span className="input-group__text">
                    <i className={iconClass}></i>
                </span>
            </div>
            <textarea {...input} placeholder={label} type={type} className={className} rows={rows}/>
            <div>
                {touched &&
                ((error && <span className={'error-message'}>{error}</span>) ||
                    (warning && <span className={'warn-Message'}>{warning}</span>))}
            </div>
        </div>:
        <React.Fragment>
            <textarea {...input} placeholder={label} type={type} className={className} rows={rows}/>
            <div>
                {touched &&
                ((error && <span className={'error-message'}>{error}</span>) ||
                    (warning && <span className={'warn-Message'}>{warning}</span>))}
            </div>
        </React.Fragment>
        }
        
    </React.Fragment>


)

