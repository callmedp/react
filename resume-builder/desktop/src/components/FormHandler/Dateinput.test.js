import { datepicker as TestDatepicker } from './formFieldRenderer'
import "../../../node_modules/react-datepicker/dist/react-datepicker.css";

import moment from 'moment';
import { shallow, mount, render } from 'enzyme';
import React from 'react';
import toJson from 'enzyme-to-json';




const defaultProps = {
    minDate: moment(0)
}

const Datepicker = (props) =>
    <TestDatepicker  {...defaultProps}
        {...props} />


describe('DatePicker', () => {

    it('render correctly date component', () => {
        const props = {
            meta: {
                touched: null,
                error: null,
                warning: null
            },
            input: {
                value: "10.03.2018"
            }
        }
        const DateInputComponent = render(<Datepicker  {...props} />);
        expect(toJson(DateInputComponent)).toMatchSnapshot();
    });

    it('render date input correctly with null value', () => {
        const props = {
            input: {
                value: null
            },
            meta: {
                touched: null,
                error: null,
                warning: null
            },
        },
            DateInputComponent = mount(<Datepicker {...props} />).find('.Error').children().at(0).getElement();
        expect(DateInputComponent.props.value).toEqual(null);
    });

    it('check the type of value', () => {
        const props = {
            meta: {
                touched: null,
                error: null,
                warning: null
            },
            input: {
                value: "10.03.2018"
            }
        }
        const DateInputComponent = mount(<Datepicker  {...props} />).find('.Error').children().at(0).getElement();
        expect(typeof DateInputComponent.props.value).toBe('string');
    });

    // it('check the onChange callback', () => {
    //     const onChange = jest.fn(),
    //         props = {
    //             input: {
    //                 value: "10.01.2018",
    //                 onChange
    //             },
    //             meta: {
    //                 touched: null,
    //                 error: null,
    //                 warning: null
    //             }
    //         },
    //         DateInputComponent = mount(<Datepicker {...props} />).find('input');
    //     console.log('-----', DateInputComponent.getElements())
    //     DateInputComponent.simulate('change', { target: { value: moment('2018-01-22').format('YYYY-MM-DD').toString() } });
    //     console.log('-----Dat  ', DateInputComponent.getElements());
    //     expect(onChange).toHaveBeenCalledWith('2018-01-21T18:30:00.000Z');
    // });

    it('check DatePicker popup open', () => {

        const
            props = {
                input: {
                    value: null
                },
                meta: {
                    touched: null,
                    error: null,
                    warning: null
                },
            },
            DateComponent = mount(<Datepicker {...props} />),
            dateInput = DateComponent.find("input[type='text']");
            
        dateInput.simulate('click');
        expect(DateComponent.find('.react-datepicker')).toHaveLength(2);
    });
})



