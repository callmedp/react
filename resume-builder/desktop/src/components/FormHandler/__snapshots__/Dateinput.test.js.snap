// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DatePicker render correctly date component 1`] = `
<div
  class="input-group "
>
  <div
    class="input-group--input-group-icon"
  >
    <span />
  </div>
  <div
    class="Error "
  >
    <div>
      <div
        class="react-datepicker__input-container"
      >
        <input
          autocomplete="off"
          class=""
          type="text"
          value="2018-10-03"
        />
      </div>
    </div>
  </div>
</div>
`;
